pkg/alertmanager/operator.go

```diff
@@ -903,6 +903,10 @@ receivers:
 		return errors.Wrap(err, "base config from Secret could not be parsed")
 	}
 
+	if err := checkAlertmanagerRootConfig(ctx, baseConfig); err != nil {
```

Comment:
could be delegated to `loadCfg` (which would be good to have renamed BTW)?

Reply:
good point